src/core/Slider/Slider.tsx

```diff
@@ -331,12 +334,15 @@ export const Slider = React.forwardRef<HTMLDivElement, SliderProps>(
 
     const handlePointerUp = React.useCallback(
       (event: PointerEvent) => {
+        if (activeThumbIndex === undefined) {
```

Comment:
When it is set to `undefined`?

Reply:
in this same function 😆 
![image](https://user-images.githubusercontent.com/9084735/151557641-cac9d966-af88-421d-a71a-a37d92a08bf6.png)